samples/boards/96b_argonkey/microphone/src/main.c

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) 2018 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <zephyr.h>
+#include <misc/printk.h>
+
+#include <board.h>
+#include <gpio.h>
+#include <led.h>
+#include <i2s.h>
+#include <i2c.h>
+#include <spi.h>
+
+#ifdef CONFIG_I2S
+#include "OpenPDMFilter.h"
+#endif
+
+#ifdef CONFIG_I2S
```

Comment:
You should use this flag here, sample has no meaning if not enabled. Rather we should add CONFIG_I2S in prj.conf if not done already.

Reply:
Yes, I added it in prj.conf for microphone sample.

Reply:
Maybe I could completely remove CONFIG_I2S in main.c.... I agree that it does not have much sense.